### ISSUE TITLE
some questiones about Git loss ( λ G )

### ISSUE THREAD

**Issue by rain2008204** [kjanjua26/Git-Loss-For-Deep-Face-Recognition#1]
@kjanjua26 @Shahnawazgrewal 

have you find some good ways to solve the long tail ?

how about subsitute embeddings for the prelogits in the network( logits = .........)

**Comment by Shahnawazgrewal** [comment]
@rain2008204 take a look at the work: [Class Rectification Hard Mining for Imbalanced Deep Learning](https://www.researchgate.net/publication/322059214_Class_Rectification_Hard_Mining_for_Imbalanced_Deep_Learning).
